regex-ballerina/Package.md

```diff
@@ -0,0 +1,8 @@
+## Module overview
+
+This module provides the following regex utility functions. 
+
+* `matches` - Checks whether a string matches a given regex.
+* `replaceAll` - Replaces every occurance of a substring that matches a given regex in a string, with another string provided.
```

Comment:
```suggestion
* `replaceAll` - Replaces every occurrence of a substring that matches a given regex in a string, with another string provided.
```

The "with another string provided" part is confusing. Can we reword this?

Reply:
Just "with another string"? Basically, it'll replace the occurrences of a substring with the provided string.